CMakeLists.txt

```diff
@@ -284,6 +284,10 @@ elseif(UNIX)
     )
     target_compile_definitions(PortAudio PUBLIC PA_USE_COREAUDIO=1)
     set(PKGCONFIG_CFLAGS "${PKGCONFIG_CFLAGS} -DPA_USE_COREAUDIO=1")
+
+    # Use C11 so that we can make use of atomic library and avoid deprecation errors.
+    target_compile_definitions(PortAudio PRIVATE std=c11)
```

Comment:
This change should also be applied to configure.in

Reply:
Yes, thanks. If this gets approved and merged for C11 then I will update configure.in.
We may also apply these changes to more targets.